core/types/interoptypes/interop.go

```diff
@@ -101,3 +101,5 @@ func TxToInteropAccessList(tx *types.Transaction) []common.Hash {
 	}
 	return hashes
 }
+
+var ErrFailsafeEnabled = errors.New("failsafe is enabled, rejecting all CheckAccessList requests")
```

Comment:
Because geth is not the entity doing the rejecting, maybe something more like "all CheckAcessList requests would be rejected" or something?